src/main/java/bookstore/repository/book/spec/DescriptionSpecificationProvider.java

```diff
@@ -0,0 +1,20 @@
+package bookstore.repository.book.spec;
+
+import bookstore.model.Book;
+import bookstore.repository.SpecificationProvider;
+import java.util.Arrays;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DescriptionSpecificationProvider implements SpecificationProvider<Book> {
```

Comment:
same, how it works? search by full Description?

Reply:
The main problem - we don't have a TS. So only imagination can stop us.
Ok, I've deleted it.